gatsbyjs/gatsby
# chore(www): Cleanup FeedbackForm

## Description

Cleanup the CSS of FeedbackForm component and remove dependency from emotion/styled

### Screenshot(s)

<img width="506" alt="feedback-form" src="https://user-images.githubusercontent.com/19193724/85910194-392c3080-b83b-11ea-863f-bdd87ae8b573.png">


### How to test

1. Go to docs page
2. The option to open feedback form will appear in the bottom right **(Was this doc helpful to you?)**

**Local URL:** http://localhost:8000/docs/
**Production URL:** https://www.gatsbyjs.org/docs/
